onnxruntime/core/providers/mkldnn/mkldnn_execution_provider.h

```diff
@@ -37,6 +39,17 @@ class MKLDNNExecutionProvider : public IExecutionProvider {
   }
 
   virtual std::shared_ptr<KernelRegistry> GetKernelRegistry() const override;
+
+  std::shared_ptr<mkldnn::memory> GetWeightMemory(std::string weightName) {
+    if (weights_mem_map.find(weightName) != weights_mem_map.end())
+      return weights_mem_map[weightName];
+    else
+      return nullptr;
+  }
+public:
+  // mkldnn formatted weights(filer data) memory from first iteration 
+  // saved by weights name
+  std::map<std::string, std::shared_ptr<mkldnn::memory>> weights_mem_map;
```

Comment:
should this be public? better to have api to Set() in addition to Get()

Reply:
Yes. We are accessing from Conv class.

Reply:
better to make it private and add an api for Set()

Reply:
sounds good.